sharing/index.md

```diff
@@ -0,0 +1,47 @@
+---
+tags:
+  - sharing
+author:
+  display_name: AGRC Office
+  email: agrc@utah.gov
+date: 2017-01-26 23:40:06
+title: Sharing GIS Data
+categories: []
+---
+
+#### Quick Links
+
+[Submitting Data to the SGID]({% link sharing/process/index.md %}) — [Basic Metadata Creation Process]({% link sharing/metadata/index.md %}) — [AGRC Sharing Policies]({% link about/policy/index.md %})
+
+## The Joys of Sharing
+
+The Utah [State Geographic Information Database (SGID)]({% link sgid/index.html%}) is one of the premier statewide GIS databases in the country, providing clean and consistent authoritative data across every geography in the state. People have been turning to the SGID for nearly 30 years to support their personal, commercial, and government projects. Our webpages have had over 30,000 views and 5,000 downloads, and the (what database?) gets over 15,000 table views a month. This would not be possible without the help and collaboration of cities, counties, and agencies across the state.
```

Comment:
**thought**: we don't have table stats for our external database at the moment. 

These number seem small and I'm not sure they are serving the purpose of creating exposure?

Reply:
They were in the previous PR and I was trying to incorporate them, knowing they probably needed to be updated.  I'd be just as happy to remove them.